PluginClasses/ZPTVACellViewController.swift

```diff
@@ -102,8 +107,61 @@ class ZPTVACellViewController : CACellViewController {
         }
     }
     
+    private func isFavoriteSupported() -> UIButton?{
+        guard let layout = self.currentComponentModel()?.style["layout_name"] as? String , let favoriteBtn  = favoritesButton, layout == "Family_5_horizontal_list_13" || layout == "Family_5_grid_11" || layout == "Family_5_grid_12"  else{
```

Comment:
cont string vars

Reply:
every let check should be on a different line

Reply:
done